omod/src/main/java/org/openmrs/web/controller/ConceptFormController.java

```diff
@@ -1036,7 +1036,8 @@ public int getNumberOfObsUsingThisConcept() {
 					name = answer.getAnswerDrug().getFullName(locale);
 				}
 				if (answer.getAnswerConcept().isRetired()) {
-					name = "<span class='retired'>" + name + "</span>";
+					String string = Context.getMessageSourceService().getMessage("Concept.retired");
```

Comment:
The name for a retired concept name needs to have a strike through, so the original code needs to be tweaked a little, may be you need to use html codes for the tags e.g **`&lt;`** for  **<**